content/02.introduction.md

```diff
@@ -41,13 +42,14 @@ We also provide critical perspectives on features that could be improved and cat
-| **merge**: Combining **commits** from two different branches together into one **branch**|
-|**release**: At any point a release can be made on GitHub to mark a significant milestone in the progression of a **repository**. While this GitHub feature is designed with releases of new versions of code in mind (_e.g._, v1.0.0), it can also be used to create a snapshot of a repository at significant stages like submission, revision, and acceptance of an associated manuscript.|
+|**branch**: Git workflow timelines or repositories are analogous to trees, with a main working project and diverging branches that are pointers to changes during the development process. A git branch is an alternative line of development for a project (repository). Branches allow users to add new features or modifications to the project without affecting the main part of the project. Development branches can be created at any point in time and work on each branch can continue independently. Branching is useful for testing out new ideas (both code and text) which may or may not eventually get integrated into the main branch of the project. Branches can also be used to isolate contributions of multiple contributors. Each person working on their own branch eliminates problems that may arise if conflicting edits are **pushed** to the same remote branch. Changes in a development branch can be **merged** into the main branch via **pull requests**. Branches can only be made by those who are given access to the project **repository**.|
+|**commit**: Commits are snapshots of the development of a project. In Git, versions of files and directories are uniquely identified as "commits", allowing one to identify and track modifications line-by-line. Commits can include changes in multiple files and must include a brief commit message describing the changes made. A typical workflow is to make some related changes in files, add a commit message (_e.g._ "Generate and include results figure"), and after several commits **push** those commits to the remote (_i.e._, cloud-based) GitHub **repository**.|
+|**push and pull**: When **commits** are made in a project locally, they must be synced with the remote GitHub repository by **pushing** them. Changes on a GitHub repository can then be **pulled** to keep your local version of the project up-to-date with the remote.|
+|**pull request**: A pull request is a request for changes made on an individual's **branch** in the repository or in a user's **fork** to be merged to the repository. Pull requests contain a description of the changes made in a **branch** or **fork** alongside all code required for testing and review by other users prior to merging into the repository.|
```

Comment:
I modified the definition of PR a little here, so make sure you agree with how it reads now. Thanks @pedrohbraga !

Reply:
It sounds fine! The only change is that I would make "testing and reviewing" in the last sentence.

Reply:
Good catch, thanks @pedrohbraga